examples/jsm/utils/RoughnessMipmapper.js

```diff
@@ -68,42 +69,27 @@ class RoughnessMipmapper {
 
 		}
 
-		if ( width !== roughnessMap.image.width || height !== roughnessMap.image.height ) {
+		const newRoughnessTexture = new FramebufferTexture( width, height, roughnessMap.format );
+		newRoughnessTexture.wrapS = roughnessMap.wrapS;
+		newRoughnessTexture.wrapT = roughnessMap.wrapT;
+		newRoughnessTexture.minFilter = roughnessMap.minFilter;
+		newRoughnessTexture.magFilter = roughnessMap.magFilter;
 
-			const params = {
-				wrapS: roughnessMap.wrapS,
-				wrapT: roughnessMap.wrapT,
-				magFilter: roughnessMap.magFilter,
-				minFilter: roughnessMap.minFilter,
-				depthBuffer: false
-			};
+		material.roughnessMap = newRoughnessTexture;
 
-			const newRoughnessTarget = new WebGLRenderTarget( width, height, params );
+		if ( material.metalnessMap == roughnessMap ) material.metalnessMap = material.roughnessMap;
 
-			newRoughnessTarget.texture.generateMipmaps = true;
+		if ( material.aoMap == roughnessMap ) material.aoMap = material.roughnessMap;
 
-			// Setting the render target causes the memory to be allocated.
+		// Copy UV transform parameters
 
-			_renderer.setRenderTarget( newRoughnessTarget );
+		material.roughnessMap.offset.copy( roughnessMap.offset );
+		material.roughnessMap.repeat.copy( roughnessMap.repeat );
+		material.roughnessMap.center.copy( roughnessMap.center );
+		material.roughnessMap.rotation = roughnessMap.rotation;
 
-			material.roughnessMap = newRoughnessTarget.texture;
-
-			if ( material.metalnessMap == roughnessMap ) material.metalnessMap = material.roughnessMap;
-
-			if ( material.aoMap == roughnessMap ) material.aoMap = material.roughnessMap;
-
-			// Copy UV transform parameters
-
-			material.roughnessMap.offset.copy( roughnessMap.offset );
-			material.roughnessMap.repeat.copy( roughnessMap.repeat );
-			material.roughnessMap.center.copy( roughnessMap.center );
-			material.roughnessMap.rotation = roughnessMap.rotation;
-			material.roughnessMap.image = roughnessMap.image;
```

Comment:
@Mugen87 I just noticed that this line got removed (I think accidentally?). It's from https://github.com/mrdoob/three.js/pull/22741 and I believe it's still necessary.

Reply:
I removed it since the image property from the original roughness texture should not overwrite the `image` property of a framebuffer texture. Both are actually different.

Sorry, I wasn't aware that my original solution did still need it.

Reply:
Yeah, if you look at the linked PR, the issue is when you go to export the  scene. USDZExporter uses the original image, and even though we're tweaking the roughness here to account for the normal map, you want the original roughness map to go through.  Also, since this only has  width and height, it just fails to export since it can't actually find an image.

Reply:
Okay, let me file a PR that adds the line back.